src/app/services/room.service.ts

```diff
@@ -1,61 +1,69 @@
 import {Injectable} from '@angular/core';
-import {Observable} from 'rxjs/Observable';
-import {HttpClient} from '@angular/common/http';
+import {HttpClient, HttpHeaders} from '@angular/common/http';
 import 'rxjs/add/observable/of';
-import {Task} from "../classes/task";
-import {$WebSocket} from "angular2-websocket/angular2-websocket";
+import {Task} from '../classes/task';
+import {$WebSocket} from 'angular2-websocket/angular2-websocket';
 
 @Injectable()
 export class RoomService {
 
-  public roomId; string;
+  public roomId: string;
   public tasks: Task[] = [];
-  public websocket = new $WebSocket("ws://plpoker-api.azurewebsites.net/websocket");
-
+  public websocket = new $WebSocket('ws://plpoker-api.azurewebsites.net/websocket');
   base = 'http://plpoker-api.azurewebsites.net';
+
   constructor(private http: HttpClient) {
   }
 
   createRoom(roomName: string) {
-    return this.http.put(this.base+'/api/Rooms',{name:roomName});
```

Comment:
do you use an transpiler like babel for the project? if so, consider using string literals, its a great way to concatenate strings

Reply:
It would be great to do so, but I am not sure if Babel supports typescript ;)